java/ingest/src/main/java/sleeper/ingest/job/IngestJobRunner.java

```diff
@@ -116,26 +116,23 @@ public IngestResult ingest(IngestJob job) throws InterruptedException, IteratorE
                         + ". This file will be ignored and will not be ingested.", pathString);
             }
         }
+        // Run the ingest
+        try (CloseableIterator<Record> recordIterator = new ConcatenatingIterator(inputIterators);
+             BufferAllocator bufferAllocator = new RootAllocator()) {
```

Comment:
This isn't creating a buffer of the configured size, as it was in IngestRecordsUsingPropertiesSpecifiedMethod.

Reply:
It also wasn't initialised at all if the configured required bytes was 0. We could create a separate method on the factory to initialise this.